subprojects/packagefiles/range-v3/meson.build

```diff
@@ -7,4 +7,7 @@ range_dep = declare_dependency(
   include_directories : range_inc
 )
 
-subdir('test')
+if get_option('tests')
+  subdir('test')
+endif
+
```

Comment:
There's stray whitespace at the end of the files you edited?

Reply:
Bad habits :man_facepalming:, fixed.